modules/configuring/proc-provisioning-your-custom-configuration.adoc

```diff
@@ -0,0 +1,189 @@
+[id="provisioning-your-custom-configuration"]
+= Provisioning your custom {product} configuration
+
+To configure {product}, provision your custom {product} config maps and secrets to {ocp-brand-name} before deploying {product}.
+
+[TIP]
+====
+For a development environment, you can install {product-short} with a default config map and secret.
+Your changes on this configuration might get reverted on {product-short} restart.
+====
+
+.Prerequisites
+* By using the link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-version}/html-single/cli_tools/index#cli-about-cli_cli-developer-commands[{openshift-cli}], you have access, with developer permissions, to the {ocp-short} cluster aimed at containing your {product-short} instance.
+* You know your desired {product-short} external URL: pass:c,a,q[{my-product-url}].
++
+[TIP]
+====
+For a development environment, consider using the {product-short} external URL that {ocp-short} provides by default.
+To identify your {product-short} external URL:
+
+. Deploy {product-short} with the default configuration.
+. From the *Developer* perspective in the OpenShift Container Platform web console, select the *Topology* view, and click the *Open URL* icon on the Developer Hub pod.
+====
+
+
+.Procedure
+. Author your custom `secrets.txt` file, to provision your secrets as environment variables values in an {ocp-short} secret, rather than in clear text in your configuration files.
```

Comment:
Do we need the comma after the 'secrets.txt file?

Reply:
```suggestion
. Author your custom `secrets.txt` file to provision your secrets as environment variables values in an {ocp-short} secret, rather than in clear text in your configuration files.
```